_helpers/dev/cli.mts

```diff
@@ -31,13 +31,20 @@ const test = program.command('test')
       .choices(['unit', 'e2e', 'all'])
       .default('all')
   )
+  .addOption(
+    new Option(
+      "-i, --image <image>",
+      "pepr controller image to use. (e.g. --image=\"pepr:dev\")",
+    ),
+  )
   .addOption(
     new Option(
       '-p, --passthru [passthru...]',
       'args to pass to test runner (e.g. --passthru=\'--testNamePattern="testName()"\')'
     )
   )
-  .action(async ({suite, passthru}) => {
+  .action(async ({suite, passthru, image}) => {
+    if(image) { process.env.IMAGE = image }
```

Comment:
```suggestion
    if (image) { process.env.PEPR_IMAGE = image }
```
Since the env vars are (essentially) a global playground, and "IMAGE" is a _very_ general term, I'm slightly worried that this is asking for some name stomping chicanery -- would you mind if we qualify our usage just a bit more...?